extensions/amp-animation/0.1/web-animations-polyfill.js

```diff
@@ -14,9 +14,21 @@
  * limitations under the License.
  */
 import {Services} from '../../../src/services';
-import {installWebAnimations} from 'web-animations-js/web-animations.install';
 
 const POLYFILLED = '__AMP_WA';
+const polyfillPromise_ = (function () {
```

Comment:
You can use `Deferred` from `src/utils/promise.js`.

Reply:
Updated to use it. The item that was a bit surprising was a `Deferred` class requires accessing the underlying `Promise` explicitly versus this version.